src/herbivoresAndCarnivores.js

```diff
@@ -1,17 +1,38 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.health = health;
+    this.name = name;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health) {
+    super(name, health);
+    this.hidden = false;
+  }
+
+  hide() {
+    this.hidden = true;
```

Comment:
```suggestion
    this.hidden = !this.hidden;
```
It should invert current value

Reply:
I can chenge it but I do not understand what diferent is between this two variants? when hidden = false => hidden = true - it will be right or no? Or I have to write hidden = !hidden?

Reply:
так ти отримуєш можливість перевикристовувати свій метод просто як перемикач стану - на протилежне незалежно від того який стан попередній

Reply:
тобто не обов'язково присвоювати значення false на початку. Треба просто ініціалізуівати змінну?

Reply:
Потрібно, щоб мати початкове значення. Але при твоєму початкувому варіанті після того як ти одного разу викличеш метод hide() - він міняє тільки на true і щоб знову міняти на false тобі треба окремий методя
Так ти викликаючи один метод можеш змінювати boolea на протилежне залежно від попереднього